app/lib/shared/utils.dart

```diff
@@ -143,6 +143,18 @@ const List<String> _reservedWords = const <String>[
   'with'
 ];
 
+const knownMixedCasePackages = const [
```

Comment:
Ran the tool

Known mixed case: Autolinker
Known mixed case: Babylon
Known mixed case: DartDemoCLI
Known mixed case: FileTeCouch
Unknown mixed case: Flutter_Nectar
Known mixed case: Google_Search_v2
Known mixed case: LoadingBox
Known mixed case: PolymerIntro
Known mixed case: Pong
Unknown mixed case: RAL
Known mixed case: Transmission_RPC
Unknown mixed case: ViAuthClient
Both mixed and lowercase: babylon

Thankfully, `Babylon` and `babylon` are owned by the same guy. He wants us to remove `Babylon`